eth2/fork_choice/src/lib.rs

```diff
@@ -44,13 +44,17 @@ extern crate types;
 
 pub mod longest_chain;
 pub mod optimised_lmd_ghost;
-pub mod protolambda_lmd_ghost;
+//pub mod protolambda_lmd_ghost;
```

Comment:
Should we just pull this stuff out if it doesn't exist? (as opposed to leaving comments)

Reply:
Only left it because I had intended of adding it in. And was a quick fix for CI. Can remove.